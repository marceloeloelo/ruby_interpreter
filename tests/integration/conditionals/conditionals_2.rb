x = 0

if x == 1
  y = 0
elsif x == 2
  y = 1
elsif x == 3
  y = 2
else
 y = 3
end

puts y # 3

if x == 1
  z = 0
elsif x == 2
  z = 1
elsif x == 3
  z = 2
end

puts z #NoMethod error z
