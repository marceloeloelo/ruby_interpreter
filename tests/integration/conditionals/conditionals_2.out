3
NameError: undefined local variable or method `z' for main:Object
